.travis.yml

```diff
@@ -6,7 +6,7 @@ services:
 language: "python"
 
 python:
-    - "3.6"
+    - "3.5"
```

Comment:
Ticket for tracking the Python 3.5 -> 3.6 change for Teamcity agents.
https://jira.mesosphere.com/browse/DCOS-19733